apps/remix-ide/src/app/tabs/locales/en/solidity.json

```diff
@@ -22,6 +22,10 @@
   "solidity.noFileSelected": "no file selected",
   "solidity.compileAndRunScript": "Compile and Run script",
   "solidity.publishOn": "Publish on",
+  "solidity.flatten": "Flatten contracts with imports before UML generation.",
```

Comment:
I think it is okay to keep only `Flatten contracts before UML generation.`

Reply:
Ok